modin/experimental/pandas/__init__.py

```diff
@@ -48,6 +48,13 @@
     to_pickle_distributed,
 )
 
+
+class ExpDataFrame(DataFrame):  # noqa: F405
```

Comment:
The new class is needed so that class modifications (for example: from `setattr` usage) do not affect the class that work in normal mode; this greatly simplifies the organization of interaction between the experimental context and the normal one.

Reply:
Outdated